src/components/search/curated/css/index.tsx

```diff
@@ -28,7 +28,7 @@ const SearchCSS = () => {
         }}
         CTAComponent={CssFormStyling}
       />
-      <div className="grid grid-cols-1 mt-8 gap-5 lg:grid-cols-3 md:grid-cols-1 sm:gap-8">
+      <div className="grid grid-cols-1 mt-8 gap-5 lg:grid-cols-3 md:grid-cols-1">
```

Comment:
removing the `sm:gap-8` fixed this. I believe it's getting the styling from a parent. I checked multiple sizes and it all seemed to work just fine.